src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Simplify kubernetes layers with New Relic
+---
+
+import k8sDiagramContext from 'images/tutorials_diagram_kubernetes-overview.webp'
+
+import tutorialOverviewdashboard from 'images/tutorial_screenshot-full_overview-dashboard.webp'
+
+import tutorialOverviewDeployments from 'images/tutorial_screenshot-full_overview-deployments.webp'
+
+import tutorialOverviewFailed from 'images/tutorial_screenshot-crop_failed-pods.webp'
+
+import tutorialAPMOverview from 'images/tutorial_screenshot-full_apmK8s.webp'
+
+import tutorialAPMPerformance from 'images/tutorial_screenshot-full_apm-k8s-performance.webp'
+
+Understanding the inner workings of your Kubernetes system, specifically how all the different functions of the system come together, is inherently complex. Let's review what makes up a Kubernetes system and then dive into how New Relic can help monitor and understand each part.
+
+## Break it apart to understand
+
+To gain a better grasp of a Kubernetes system, let's break it down into distinct layers and explore how New Relic can help you comprehend each one effectively.
+
+<img
+    title="k8sDiagramContext"
+    alt="an image showing an abstracted view of a kubernetes system. This includes pods, apps, and clusters"
+    src={k8sDiagramContext}
+/>
+
+We'll discuss a Kubernetes system in three key sections:
+
+  * **The Cluster**: This represents the entire Kubernetes system. Imagine a vast bakery comprised of many ovens each with its own set of bakers. The cluster contain multiple deployments, which in turn house many pods, with each pod containing its individual services and applications.
```

Comment:
The capitalization feels weird to me in these three bullets...

Reply:
Although I sort of like the bakery metaphor, I'm not sure it totally works. I think you may need to it more explicitly to the different Kubernetes things. Is a pod an oven? What is the baker? Is it a deployment? Is a cluster an oven? See, it gets kind of tangled unless you get super specific.